### PR TITLE
Baking STI scripts into the image

### PR DESCRIPTION
**Pull request by jhadvig**

### CODE REVIEW

**Comment by jhadvig** [comment]
[test]

File: 0.10/Dockerfile [hunk 1]
```diff
@@ -41,4 +38,4 @@ USER nodejs
 
 EXPOSE 3000
 
-CMD ["/opt/nodejs/bin/usage"]
\ No newline at end of file
+CMD ["/usr/local/bin/usage"]
```
Comment (mfojtik) [inline 0.10/Dockerfile]: This should be /usr/local/sti/ ?

Also if you add this into PATH, you don't have to write absolute path.

**Comment by bparees** [comment]
lgtm

**Comment by bparees** [comment]
appears our test bot is letting us down again.

**Comment by jhadvig** [comment]
@bparees @mfojtik I've updated the PR with commit that adds an extra test case, which tests if usage script path is set correctly in the Dockerfile.
Thoughts ?

**Comment by mfojtik** [comment]
@jhadvig can you get rid of ENV APP_ROOT . altogether?

**Comment by mfojtik** [comment]
@jhadvig also if the WORKDIR is set to application runtime directory, make sure you nuke all absolute paths from all STI scripts.

**Comment by bparees** [comment]
@jhadvig new test case looks good

**Comment by jhadvig** [comment]
@bparees @mfojtik I've updated the commits and also add one which replaces `centos7` image with `openshift/base-centos7`.
Can you please take a look ?

**Comment by bparees** [comment]
lgtm, but reminds me that sooner or later we need a rhel7 base.

**Comment by mfojtik** [comment]
LGTM

**Pull request merged**